Sources/MapboxNavigation/Constants.swift

```diff
@@ -40,6 +40,11 @@ public let CongestionAttribute = "congestion"
  */
 public let NavigationViewMinimumVolumeForWarning: Float = 0.3
 
+/**
+ The distance of blending color between two different congestion level segment in meters.
+ */
+public var GradientCongestionBlendingDistance: Double = 30.0
```

Comment:
We should use the same term throughout this PR: is it fading or blending?

Since this is a physical distance measured in meters, the type should be `CLLocationDistance`.

Should this value be configurable in a Style?

Reply:
I make it as a global variable like the `RouteControllerProactiveReroutingInterval `, they seems very similar in changing the refresh rate, or the color fading distance.
I'll set it as `CLLocationDistance ` and improve the description term.